bindings/node.js/binding.gyp

```diff
@@ -2,27 +2,44 @@
   "targets": [
     {
       "target_name": "kzg",
-      "cflags!": ["-fno-exceptions"],
-      "cflags_cc!": ["-fno-exceptions"],
-      "xcode_settings": {
-        "CLANG_CXX_LIBRARY": "libc++",
-        "MACOSX_DEPLOYMENT_TARGET": "13.0"
-      },
+      "sources": [
+        "src/kzg.cxx",
+        "deps/blst/src/server.c",
+        "deps/c-kzg/c_kzg_4844.c"
+      ],
       "defines": [
         "NAPI_DISABLE_CPP_EXCEPTIONS",
-        "FIELD_ELEMENTS_PER_BLOB=<!(echo ${FIELD_ELEMENTS_PER_BLOB:-4096})"
+        "FIELD_ELEMENTS_PER_BLOB=4096"
       ],
-      "sources": ["src/kzg.cxx"],
       "include_dirs": [
-        "../../inc",
-        "../../src",
+        "<(module_root_dir)/deps/blst/bindings",
+        "<(module_root_dir)/deps/c-kzg",
         "<!@(node -p \"require('node-addon-api').include\")"
       ],
-      "libraries": [
-        "<(module_root_dir)/c_kzg_4844.o",
-        "<(module_root_dir)/../../lib/libblst.a"
-      ],
-      "dependencies": ["<!(node -p \"require('node-addon-api').gyp\")"]
+      "conditions": [
+        ["OS!='win'", {
+          "sources": ["deps/blst/build/assembly.S"],
+          "cflags_cc": [
+            "-std=c++17",
+            "-fPIC"
+          ]
+        }],
+        ["OS=='mac'", {
+          "xcode_settings": {
+            "CLANG_CXX_LIBRARY": "libc++",
+            "MACOSX_DEPLOYMENT_TARGET": "13.0"
+          }
+        }],
+        ["OS=='win'", {
+          "defines": ["_CRT_SECURE_NO_WARNINGS"],
```

Comment:
Does this still need to be here if its in `src/Makefile` now?

Reply:
Yes, I believe so. The makefile isn't being used in this process. I'll experiment with removing stuff later.